src/services/auth.js

```diff
@@ -111,6 +111,7 @@ class Auth {
     const isExpired = now > decoded.exp;
 
     if (isExpired && !refreshToken) {
+      window.dispatchEvent(new CustomEvent('MSSDK:token-expired'));
```

Comment:
this function (isLogged) is executed in a few places in our app but it doesn't have a whole refresh logic and it's never executed with the API requests. I think it's should not be added here